deno/lib/types.ts

```diff
@@ -486,6 +500,19 @@ export abstract class ZodType<
     });
   }
 
+  exemplify(example: unknown): this {
```

Comment:
Having type safety for examples seems like quite a important feature.
```ts
exemplify(example: ParseReturnType<this["_output"]>): this {
```

Or something similar would ensure that required changes to examples are caught if the definition changes.